frontend/src/hooks/useChat.tsx

```diff
@@ -1,10 +1,10 @@
-import { Session } from '@vonage/client-sdk-video';
 import { RefObject, useCallback, useState } from 'react';
 import useUserContext from './useUserContext';
 import { ChatMessageType } from '../types/chat';
+import VonageVideoClient from '../utils/VonageVideoClient';
 
 export type UseChatProps = {
-  sessionRef: RefObject<Session | null>;
+  sessionRef: RefObject<VonageVideoClient | null>;
```

Comment:
Same nit here.  Chat just requires Session APIs so passing in All Vonage APIs seems like overkill.